agent/consul/health_endpoint.proto

```diff
@@ -0,0 +1,15 @@
+syntax = "proto3";
+
+package consul;
+
+service Health {
```

Comment:
Is this just a testing stub? Maybe we could name it that way then? the real endpoint is just the Consul service now right?

Reply:
I thought this was removed, it looks like it came back during rebasing somehow